packages/services/src/balances/account-balances.service.ts

```diff
@@ -0,0 +1,56 @@
+import { injectable } from 'inversify';
+
+import { Money } from '@leather.io/models';
+import { isDefined, sumMoney } from '@leather.io/utils';
+
+import { AccountRequest } from '../types';
+import { BtcBalancesService } from './btc-balances.service';
+import { RunesBalancesService } from './runes-balances.service';
+import { Sip10BalancesService } from './sip10-balances.service';
+import { StxBalancesService } from './stx-balances.service';
+
+@injectable()
+export class AccountBalancesService {
+  constructor(
+    private readonly btcBalancesService: BtcBalancesService,
+    private readonly stxBalancesService: StxBalancesService,
+    private readonly sip10BalancesService: Sip10BalancesService,
+    private readonly runesBalancesService: RunesBalancesService
+  ) {}
+
+  public async getTotalBalance(request: AccountRequest, signal?: AbortSignal): Promise<Money> {
```

Comment:
@alexp3y I was thinking about this last week and I wonder if we **need** two queries here. 

`useGetAccountTotalBalanceQuery` and `useGetAccountUnlockedBalanceQuery` are basically the same except we return `stxBalance.quote.availableBalance` for `total` and `stxAccountBalance.value?.quote.availableUnlockedBalance` for `unlocked` 

Unless I am missing something, we could just have one request and return a both `total` and `unlocked`?